packages/odyssey-react-mui/src/Tabs.tsx

```diff
@@ -53,16 +54,21 @@ export type TabsProps = {
    */
   ariaLabel?: string;
   /**
-   * The value of the Tab that should be selected by default
+   * The value of the Tab that should be selected by default.
+   * If value is provided as well, value takes precedence.
```

Comment:
```suggestion
   * The value of the Tab that should be selected by default.
   * When `value` is provided, it takes precedence.
```

Can we also mark `initialValue` as `@deprecated`? Seems like it doesn't need to be here.

It's completely redundant especially because MUI only takes `value`. That was a good point you made.